cmd/tendermint/commands/run_node.go

```diff
@@ -102,6 +105,9 @@ func NewRunNodeCmd(nodeProvider nm.Provider) *cobra.Command {
 		Use:     "start",
 		Aliases: []string{"node", "run"},
 		Short:   "Run the tendermint node",
+		Long: `Run the tendermint node
+
+need to set "--mode validator" to run the node as validator`,
```

Comment:
I do no think this additional comment is necessary. If we want to describe mode we should have a more elaborate description of all the possible values.